docs/contributing-admins.rst

```diff
@@ -0,0 +1,53 @@
+:orphan:
+
+Instructions for aiohttp admins
+===============================
+
+This page is intended to document certain processes for admins of the aiohttp repository.
+For regular contributors, return to :doc:`contributing`.
+
+.. contents::
+   :local:
+
+Creating a new release
+----------------------
+
+To create a new release:
+
+#. Start on the branch for the release you are planning (e.g. ``3.8`` for v3.8.6): ``git checkout 3.8 && git pull``
+#. Update the version number in ``__init__.py``.
+#. Run ``towncrier``.
+#. Check and cleanup the changes in ``CHANGES.rst``.
+#. Checkout a new branch: e.g. ``git checkout -b release/v3.8.6``
+#. Commit and create a PR. Once PR is merged, continue.
+#. Go back to the release branch: e.g. ``git checkout 3.8 && git pull``
+#. Add a tag: e.g. ``git tag -a v3.8.6 -m 'Release 3.8.6'``
+#. Push the tag: e.g. ``git push origin v3.8.6``
```

Comment:
If this assumes that `origin` is the upstream repo, we should document it. Some people use `fork` for their forks and `origin` for upstream, while others use `upstream` for the upstream and `origin` for their fork. Also, there's people using their account name for their fork (as in `username` for `https://github.com/username/aiohttp.git`).

It's a good idea to clearly declare such expectations.

Here's an example of a such doc: https://docs.ansible.com/ansible-core/devel/community/development_process.html#backporting-merged-prs-in-ansible-core. Here's one project using the `upstream` convention: https://github.com/python-attrs/attrs/blob/bd443b0/.github/CONTRIBUTING.md#local-development-environment.